src/docGenerator.ts

```diff
@@ -88,14 +89,26 @@ export class DocGenerator {
         category,
         subcategory,
         DocGenerator.generateParameters(patternId, meta?.schema),
-        meta?.docs?.recommended === true
+        this.enablePattern(patternId, meta)
       )
     })
 
     return new Specification(toolName, toolVersion, patterns)
   }
 
-  generateDescriptionEntries (): DescriptionEntry[] {
+  private enablePattern (patternId: string, meta: Rule.RuleMetaData): boolean {
+    const listPrefix = [
+      "@typescript-eslint",
+      "eslint-plugin",
+      "security",
+      "security-node"
+    ]
+    const prefix = patternId.split("/")[0]
+    
+    return meta?.docs?.recommended && (prefix === patternId || listPrefix.includes(prefix))
```

Comment:
Why do we need `prefix === patternId`? Shouldn't be enough to look for the prefixes on the listPrefix list?